Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -24,7 +24,6 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Web" />
-    <Reference Include="System.Runtime" />

```

Comment:
Basically remove this one. Since this reference is not used in .net46 at all. I am using `System.Runtime.InteroServices` in .net standard 2.0 project to get runtime info.